controllers/module_reconciler.go

```diff
@@ -262,6 +263,27 @@ func (r *ModuleReconciler) handleDevicePlugin(ctx context.Context, mod *ootov1al
 	return err
 }
 
+func (r *ModuleReconciler) getRequestedModule(ctx context.Context, namespacedName types.NamespacedName) (*ootov1alpha1.Module, error) {
+	logger := log.FromContext(ctx)
+
+	mods := ootov1alpha1.ModuleList{}
+
+	err := r.Client.List(context.Background(), &mods)
+	if err != nil {
+		logger.Error(err, "could not list modules")
+		return nil, err
+	}
+
+	r.metric.SetExistingKMMOsCRs(len(mods.Items))
```

Comment:
```suggestion
	r.metric.SetExistingKModuleCRs(len(mods.Items))
```

Reply:
isn't KMMO better? That's what we call the operator, so it should handle KMMO CRs. Not sure what does it mean KModule

Reply:
In `module_reconciler.go` you do `err := r.Client.List(context.Background(), &mods)` which mean you only list `Modules` resources. We may have more CustomResources in the future (preflight for example).

By using `KMMOsCRs` I expect it to count all CRs (not only modules) but this is not what the code you added is doing so I think it is missleading don't you think so?!

Reply:
preflight won't be a Module CR , it will be a different one, so that won't be a problem